release_notes/topics/17_0_0.adoc

```diff
@@ -0,0 +1,11 @@
+= Highlights
+
+== Improved User Search
+
+Keycloak now supports a syntax for the user search, which allows for three different types of searches :
```

Comment:
It would be good to include information on the specific admin UI screens that were affected, e.g.
```suggestion
{project_name} now supports a glob-like syntax for the user search when listing users in admin console, which allows for three different types of searches: prefix (`foo*` which became default search), infix (`*foo*`), and exact `"foo"`)
```

Reply:
Done. Removed the bullets points.